nixos/modules/services/monitoring/icinga2.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.icinga2;
+  environment = pkgs.writeText "icinga-env" ''
+    DAEMON=${pkgs.icinga2}/sbin/icinga2
+    ICINGA2_CONFIG_FILE=/etc/icinga2/icinga2.conf
+    ICINGA2_RUN_DIR=/var/run
+    ICINGA2_STATE_DIR=/var/icinga2
```

Comment:
Should be /var/lib/icinga2 to be in line with other packages.

Reply:
What does upstream here? This should be the default otherwise there is an important reason.

Reply:
Most of the NixOS stuff ends up in ```/var/lib/[package]``` so if all we need to do is define that variable, shouldn't we choose to be consistent with ourselves instead of just following upstream?

Reply:
Then we break documentation and third party-tools, which I find a more important goal then the aesthetics of consistency.

Reply:
It's /var/lib/icinga2 on centos 7 icinga instance I have, I'm changing it here.